src/argy-bargy.janet

```diff
@@ -20,7 +20,11 @@
   []
   (if (nil? cols)
     (do
-      (def p (os/spawn ["tput" "cols"] :p {:out :pipe :err :pipe}))
+      (def cmd
+        (if (= :windows (os/which))
+          ["powershell" "-command" `&{(get-host).ui.rawui.WindowSize.Width;}`]
```

Comment:
Is there any reason to have the final string in `` ` `` rather than `"`?

Reply:
Sorry, no.

There used to be double quotes inside.

I'll replace the backticks with double quotes.